esx_service/utils/kvESX.py

```diff
@@ -31,6 +31,7 @@
 
 # VSphere lib to access ESX proprietary APIs.
 DISK_LIB = "/lib/libvmsnapshot.so"
+DISK_LIB64 = "/lib64/libvmsnapshot.so"
```

Comment:
Can you elaborate on how the loading of correct versions of esx  and our Libs supposed to work end to end, ? Are  you saying in esx 6.5 the 32 bit lib is missing and you use it for determine the bitness ?